value.go

```diff
@@ -5,7 +5,8 @@ type Value interface {
 	GetID() string
 	GetCreatedAt() int64
 	GetUpdatedAt() int64
-	GetRelationshipIDs() []string
```

Comment:
Why the signature change? Can we still support this function w/ it's old signature at the same time, or is this backwards incompatible change required?

Reply:
Great question! I left a comment change explaining why :)